packages/altair/test_altair.py

```diff
@@ -3,7 +3,7 @@
 
 
 @pytest.mark.xfail_browsers(firefox="slow")
-@run_in_pyodide(packages=["altair"])
+@run_in_pyodide(packages=["altair", "pandas"])
 def test_altair(selenium):
     import altair as alt
     import pandas as pd
```

Comment:
This can fail in some cases (as it did in this instance) because `pandas` was removed as a runtime dependency above, so there is a chance `pandas` will not have been built by then (or will have been in the process of building in a different CI job). I am not sure if there is a reliable way to test both packages without one depending on another, do you have any thoughts, @ryanking13?

Reply:
Thanks for that info! Technically, we don't need to use pandas and can create chart data via altair directly. I switched to using one of these approaches since it seems like an easier way to get this test to work.

Reply:
You can use a similar approach [we use in duckdb test](https://github.com/pyodide/pyodide/blob/a5386c86b86135bda9c6f5d2d4f214726bacb7af/packages/duckdb/test_duckdb.py#L16); skipping the test until the dependency is built. But updating the test sounds better.